index.html

```diff
@@ -173,7 +226,7 @@ <h2>Destinations</h2>
         </table>
       </div>
     </div>
-    <div class="row my-5 d-flex justify-content-center "> <!-- JavaScript Component: Collapse -->
+    <div class="row my-5 d-flex justify-content-center "> <!-- JavaScript Component: Collapse + Tooltip -->
         <p>
             <a class="btn btn-secondary" data-toggle="collapse" href="#multiCollapseExample1" role="button" aria-expanded="false" aria-controls="multiCollapseExample1">Fun In The Sun</a>
```

Comment:
This is an `a` element instead of a button like the rest, thus it doesn't collapse properly.